HandheldCompanion/Views/Windows/OverlayQuickTools.xaml.cs

```diff
@@ -120,33 +140,113 @@ public OverlayQuickTools()
         _pages.Add("QuickSuspenderPage", suspenderPage);
     }
 
-    public static OverlayQuickTools GetCurrent()
+    private void PlatformManager_CPULoadChanged(float? value)
     {
-        return CurrentWindow;
+        if (value == null)
+            return;
+
+        Application.Current.Dispatcher.BeginInvoke(() =>
+        {
+            var CPULoadPercent = (int)Math.Truncate((float)(value));
```

Comment:
Shouldn't be done once it's using MVVM principle.

Reply:
Yeah this was done like that because i had a look at examples i saw on other pages